packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -90,11 +122,8 @@
 	display: grid;
 	grid-template-columns: 1fr;
 	gap: $grid-unit-40;
-	// Small top padding required to avoid cutting off the visible outline
-	// when hovering items.
-	padding-top: $border-width-focus-fallback;
```

Comment:
In this PR, the pattern list has enough padding so this small padding should be unnecessary.